static/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -120,9 +120,24 @@ const newIssueQuery = {
   orderby: '',
 };
 
-const DATASET_CHOICES: [WidgetType, string][] = [
-  [WidgetType.DISCOVER, t('All Events (Errors and Transactions)')],
-  [WidgetType.ISSUE, t('Issues (States, Assignment, Time, etc.)')],
+const newMetricsQuery = {
+  name: '',
+  fields: ['sum(session)'] as string[],
```

Comment:
This is the correct field for sessions API. 
In `/metrics` endpoint, this field is prefixed to avoid any naming collisions with future custom metrics.
```suggestion
  fields: ['sum(sentry.sessions.session)'],
```

Reply:
good catch! in this case, should we use `SessionMetric.SENTRY_SESSIONS_SESSION`?

https://github.com/getsentry/sentry/blob/eb5781c217833acf605cbf4ec7de5e39b6756614/static/app/utils/metrics/fields.tsx#L5